app/Jobs/ElasticSearch/ForceSearchIndex.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Jobs\ElasticSearch;
+
+use Illuminate\Support\Facades\Log;
+
+/**
+ * 
+ * Job for running CirrusSearch/ForceSearchIndex.php on a wiki
+ * 
+ * Example:
+ * 
+ * php artisan job:dispatchNow ElasticSearch\\ForceSearchIndex 1 0 1000 ,
+ */
+class ForceSearchIndex extends CirrusSearchJob
+{
+    private $fromId;
+    private $toId;
+
+    public function __construct( int $wikiId, int $fromId, int $toId  ) {
```

Comment:
It would be great to make this consistent with some of the other jobs that allow selecting by domain etc.
This will especially be handy for the migration

Reply:
done